src/main/java/io/confluent/connect/hdfs/wal/FSWAL.java

```diff
@@ -102,7 +102,21 @@ public void apply() throws ConnectException {
       if (!storage.exists(logFile)) {
         return;
       }
+
```

Comment:
Just to check -- is the assumption here that even if we `apply()` and effectively do nothing, even if another process exists and then subsequently writes to the WAL, not acquiring the WAL lease is ok because we will acquire it later when writing and see the conflict?